vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -176,12 +176,17 @@ See the [Plugin Environment Variables](../../tutorials/devops/plugin-environment
     },
     {
       "name": "my-mfe2",
-      "stack": "react",
+      "stack": "custom",
       "type": "widget",
       "publicFolder": "public",
       "titles": { "en": "My MFE2 Title", "it": "Il Mio Titolo MFE2" },
       "group": "free",
-      "commands": { "build": "custom-command" }
+      "commands": {
+           "build": "npm run build",
+           "run": "npm install -g http-server",
+            "pack": "docker build ..."
```

Comment:
@nshaw, can you correct this? I tried to add an example and could not find an equivalent for the 3 commands for build, run, and pack. Maybe this is unnecessary.. LMK

Reply:
@jyunmitch I might use the echo examples from https://entando.myjetbrains.com/youtrack/issue/ENG-4175. That helps communicate the 3 commands that can/should be customized but we don't have to determine realistic commands for an unknown stack.